spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :feature do
+  let(:user) { create(:user) }
+  let(:post) { create(:post, author: user) }
+  let!(:comments) do
+    create_list(:comment, 3, post:, user:)
+  end
+
+  before do
+    visit user_post_path(user, post)
+  end
+
+  it 'shows the post details with comments' do
+    expect(page).to have_content(post.title)
+    expect(page).to have_content(post.text)
+    expect(page).to have_content("Comments: #{post.comments.size}")
+    expect(page).to have_content("Likes: #{post.likes_counter}")
```

Comment:
- [ ] This test 👇 is not yet added to your implementation, kindly attend to it.  👍 

<img width="291" alt="Untitled" src="https://github.com/shm04/blogApp/assets/103386183/8e4229f5-2428-4c29-b767-5277fdafbe0d">